ci-operator/step-registry/telcov10n/functional/cnf-core/telcov10n-functional-cnf-core-workflow.yaml

```diff
@@ -0,0 +1,9 @@
+workflow:
+  as: telcov10n-functional-cnf-core
+  steps:
+    pre:
+    - ref: telcov10n-functional-ocp-baremetal-ipi-deployment
+    - ref: telcov10n-functional-ocp-olm-setup
+    - ref: telcov10n-functional-ocp-cnf-core-tests
```

Comment:
This should be removed as it exists in the config itself (I added a comment there)

Reply:
It should be run twice: once for cluster preparation and a second time for the testing itself

Reply:
Why do you need to run the tests in the cluster preparation step? are there any changes in the cluster so you need to run the tests twice?